server/service/apple_mdm.go

```diff
@@ -3398,6 +3446,10 @@ func SendPushesToPendingDevices(
 	commander *apple_mdm.MDMAppleCommander,
 	logger kitlog.Logger,
 ) error {
+	// TODO(iosrevive): (optional), if we want to go out of our way to re-create
+	// the deleted iDevices ASAP, we could include them in the list of hosts to
+	// send a push to, so that they checkin early (it can be arbitrarily long
+	// without a push notification).
```

Comment:
Pending product decision (asked on Slack here: https://fleetdm.slack.com/archives/C03C41L5YEL/p1742413143321909)

Reply:
Seems like we want it so I'll add to this PR next week to finish this up.